### PR TITLE
Archi sandbox

### LINKED ISSUE #6: Support OKIB in multiple text channels
Maybe this already 100% works in different channels. However, I would like to see support for independent (simultaneous/non-interfering) OKIB gathers in multiple channels, which is definitely not supported. I think `pub-games` channel or `inhouse-high-diff` channel can be dedicated to BNet gathers, to get NA/bnet ppl more M+ practice.

### PR DESCRIPTION
**Pull request by Archimonde666**
Pub host !okib enable

### CODE REVIEW

**Comment by Archimonde666** [comment]
@jmrico01 what did i do, i retrieved all file from your last commit on master branch and now i feel like everything has resetted

help

File: main.py [hunk 1]
```diff
@@ -526,6 +526,10 @@ async def on_message(message):
 OKIB_GATHER_EMOJI_STRING = "<:ib:{}><:ib2:{}>".format(IB_EMOJI_ID, IB2_EMOJI_ID)
 OKIB_GATHER_PLAYERS = 8 # not pointless - sometimes I use this for testing
 
+PUB_HOST_ROLE_ID = 791279611311947796
```
Comment (jmrico01) [inline main.py:529]: ugh, ideally these would go into the `params.py`, but I know that's annoying for you to edit. can change later I guess, whatever.

File: main.py [hunk 2]
```diff
@@ -688,7 +698,11 @@ async def noib(ctx):
     global _okib_channel
     global _okib_message_id
     
-    if ctx.message.author.roles[-1] <= _guild.get_role(params.PEON_ID):
+    #PUB OKIB
+    if ctx.channel ==  _bnet_channel:
+        pass
```
Comment (jmrico01) [inline main.py:703]: I think this means anyone can !noib in the pub channel. That's not what we want, right?

File: main.py (hunk 1 above)
Comment (jmrico01) [reply main.py:529]: yeah nvm I'll rethink this later, ignore this comment

File: main.py (hunk 2 above)
Comment (Archimonde666) [reply main.py:703]: Not at all, it just bypass the peon or inferior check (which couldn't noib anyway) if we're in the pub channel.
THEN
Line 708 check still play his role => only gatherer, or shaman can !noib it in the first 2 hours.
Effectively after 2 hours anyone can !noib it. Do we still want to limit that type of !noib to a certain role or @everyone is fine with that ?

Comment (jmrico01) [reply main.py:703]: oooh you're right, mb. Ok, I think maybe limit !noib to pub hosts or above just to keep it consistent with the !okib permissions, like on line 625

Comment (Archimonde666) [reply main.py:703]: Alr, added that in line 702 if it's not satisfied it'll end up with regular peon check and reject the non pub host or above fool

Comment (jmrico01) [reply main.py:703]: lol, funny way to do it but sounds good

**Review by jmrico01 — APPROVED** [review]

**Pull request merged**